nbtc/relayer.go

```diff
@@ -20,12 +21,21 @@ type Relayer struct {
 	btcClient        bitcoin.Client
 	shutdownChan     chan struct{}
 	processTxsTicker *time.Ticker
+	confirmTxsTicker *time.Ticker
 }
 
 // NewRelayer creates a new Relayer instance with the given configuration.
-func NewRelayer(btcClientConfig rpcclient.ConnConfig, processTxsInterval time.Duration, db *dal.DB) (*Relayer, error) {
+func NewRelayer(
+	btcClientConfig rpcclient.ConnConfig,
+	processTxsInterval time.Duration,
+	confirmTxsInterval time.Duration,
+	db *dal.DB,
+) (*Relayer, error) {
+
 	if db == nil {
-		return nil, fmt.Errorf("database cannot be nil")
+		err := fmt.Errorf("database cannot be nil")
+		log.Err(err).Msg("")
+		return nil, err
 	}
 
 	if btcClientConfig.Host == "" || btcClientConfig.User == "" || btcClientConfig.Pass == "" {
```

Comment:
We can call this the default config and create the function `DefaultConfig()` to return this struct.

Reply:
I am not sure this is neccesary since we will be using it once

Reply:
+ we already have the defult values in the constructor